python/ray/util/client/worker.py

```diff
@@ -188,7 +215,6 @@ def _get(self, ref: ClientObjectRef, timeout: float):
             except pickle.UnpicklingError:
                 logger.exception("Failed to deserialize {}".format(data.error))
                 raise
-            logger.error(err)
```

Comment:
Is this intentional?

Reply:
Yeah, it was duplicating the error in the console otherwise

Reply:
Ahh okay :)